web/modules/custom/dpl_react_apps/dpl_react_apps.module

```diff
@@ -49,3 +49,17 @@ function dpl_react_apps_preprocess_page(array &$variables): void {
   ];
 
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ *
+ * @param mixed[] $variables
+ *   The variables for the theme hook.
+ */
+function dpl_react_apps_preprocess_dpl_react_app(array &$variables): void {
+  // Since we need these texts in all the apps we add them here.
+  $variables['attributes'] += [
+    'data-alert-error-close-text' => t('Close', [], ['context' => 'React apps global: Error handling']),
+    'data-alert-error-message-text' => t('An error occurred', [], ['context' => 'React apps global: Error handling']),
+  ];
```

Comment:
```suggestion
  $variables['attributes'] += [
    'data-alert-error-close-text' => t('Close', [], ['context' => 'React apps (Global error handling)']),
    'data-alert-error-message-text' => t('An error occurred', [], ['context' => 'React apps (Global error handling)']),
  ];
```

As I see it we are struggling a bit with subsections of contexts. [Other places I see us using `([subsection])` to delimit subsections such as ARIA](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/e314165d6278cd4624e45c3c92e289a832b55a84/web/modules/custom/dpl_fees/src/Plugin/Block/FeesListBlock.php#L79). For the sake of consistency this suggests continuing with that pattern instead of doing something else.

Reply:
Good point 👍